Update Regex, Cache, File, Xmldata, Task and Xslt BBEs (#2954)
ballerina-platform/ballerina-distribution: examples/manage-scheduled-jobs/manage_scheduled_jobs.md

# Manage sheduled jobs

> The `task` library provides functions to manage the scheduled jobs such as pause, resume, unschedule, and etc.

Reviewer: ```suggestion
The `task` library provides functions to manage the scheduled jobs such as pause, resume, unscheduled, etc.
```

> For more information on the underlying module, see the [`Task` module](https://lib.ballerina.io/ballerina/task/latest/).

Reviewer: ```suggestion
For more information on the underlying module, see the [`task` module](https://lib.ballerina.io/ballerina/task/latest/).
```

::: code manage_scheduled_jobs.bal :::

> To run this sample, use the `bal run` command.

Reviewer: ```suggestion
To run this sample, use the `bal run` command.
```
Same as above.
